web/vtadmin/src/components/NavRail.module.scss

```diff
@@ -117,3 +117,24 @@ a.navLink:focus .icon,
 a.navLink:hover .icon {
     fill: var(--colorPrimary);
 }
+
+.hotkey::before {
+    $hotkeySize: 16px;
+
+    @apply rounded-sm;
```

Comment:
I never really know where to draw the line between `@apply` and just writing out CSS rules with variables, so open to suggestions here!

Reply:
Me neither! I'd only say that since we have tailwind, I'd personally use it to its ✨ fullest ✨